src/installers/prisma.ts

```diff
@@ -41,21 +37,10 @@ export const prismaInstaller: Installer = async ({
   const packageJsonContent = fs.readJSONSync(packageJsonPath) as PackageJson;
   packageJsonContent.scripts!.postinstall = "prisma generate"; //eslint-disable-line @typescript-eslint/no-non-null-assertion
 
-  await Promise.all([
-    fs.copy(schemaSrc, schemaDest),
-    fs.copy(clientSrc, clientDest),
-    fs.copy(sampleApiRouteSrc, sampleApiRouteDest),
-    fs.writeJSON(packageJsonPath, packageJsonContent, {
-      spaces: 2,
-    }),
-  ]);
-
-  // only generate client if we have installed the dependencies
-  if (!noInstall) {
-    const generateClientCmd =
-      pkgManager === "npm"
-        ? "npx prisma generate"
-        : `${pkgManager} prisma generate`;
-    await execa(generateClientCmd, { cwd: projectDir });
```

Comment:
this will be executed in post install script anyway so no need to run it here too